### ISSUE TITLE
Do not remove consecutive spaces in the REPL

### ISSUE THREAD

**Issue by sorawee** [brownplt/pyret-lang#575]
```
> "a a"
"a a"
> "a             a"
"a a"
```

The output string is misleading. I expect the output to be the same as what users input.

**Comment by awstlaur** [comment]
Also this 

```
test ("a a" is "a     a"): failed, reason:
Values not equal

"a a"

"a a"
```

**Comment by schanzer** [comment]
Yikes. That's going to be hard to explain to kids!

**Comment by blerner** [comment]
Joe, is this just a "white-space: pre" styling issue?

**Comment by jpolitz** [comment]
Thanks.  Fixed by https://github.com/brownplt/code.pyret.org/commit/7618e3232df7aff4ecbe995e35b888bb2b47fbd8

**Comment by blerner** [comment]
Thinking about it some more, I vaguely recall *removing* "white-space: pre" at some point, because otherwise really long values wouldn't line-wrap in the REPL, and scrolling was a pain.  Maybe we want this to be "pre-wrap", then?  Or maybe it's no longer an issue with the richer-reprs (but I doubt it)...

**Comment by jpolitz** [comment]
CSS aside, what do we want folks to see in all these cases?

**Comment by jpolitz** [comment]
The immediate problem (though not the wrapping problem) is fixed by 7618e3232df7aff4ecbe995e35b888bb2b47fbd8

**Comment by blerner** [comment]
I think this is fixed; if wrapping is a problem later, let's file another issue.
